Misc/NEWS.d/next/Core and Builtins/2019-10-10-14-04-26.bpo-38425.xB59iJ.rst

```diff
@@ -0,0 +1,2 @@
+Remove ‘res’ may be used uninitialized warning. (Contributed by Dong-hee Na
+in :issue:`38425`.)
```

Comment:
Please remove the NEWS entry. Fixing a compiler warning doesn't deserve to be notificed to users.